### PR TITLE
Feat: enhance draw circle

### PR DESCRIPTION
**Pull request by mbarbet**

### CODE REVIEW

**Review by sebbousquet — APPROVED** [review]

File: projects/arlas-components/src/lib/components/mapgl/draw/draw.service.ts [hunk 1]
```diff
@@ -226,7 +227,7 @@ export class MapboxAoiDrawService {
     const wh = this.calculateEnvelopeDimension(feature);
     this.editAoiSource.next({
       area: a,
-      areaMessage: a > 0 ? '': 'Draw at least 2 points.',
+      areaMessage: a > 0 ? '' : 'Draw at least 2 points.',
```
Comment (QuCMGisaia) [inline projects/arlas-components/src/lib/components/mapgl/draw/draw.service.ts:230]: Add a marker here to translate the message

File: projects/arlas-components/src/lib/components/mapgl/draw/modes/circles/circle.mode.ts [hunk 2]
```diff
@@ -32,6 +32,8 @@ circleMode.onSetup = function (opts) {
 
     return {
         initialRadiusInKm: opts.initialRadiusInKm || DEFAULT_RADIUS_IN_KM,
+        steps: opts.steps || 64,
+        units: opts.units || 'kilometers',
```
Comment (QuCMGisaia) [inline projects/arlas-components/src/lib/components/mapgl/draw/modes/circles/circle.mode.ts:36]: marker on the kilometers

File: projects/arlas-components/src/lib/components/mapgl/draw/modes/circles/radius.circle.mode.ts [hunk 3]
```diff
@@ -106,6 +108,8 @@ radiusCircleMode.onSetup = function (opts) {
     return {
         ...props,
         circle:polygon,
+        steps: opts.steps || 64,
+        units: opts.units || 'kilometers'
```
Comment (QuCMGisaia) [inline projects/arlas-components/src/lib/components/mapgl/draw/modes/circles/radius.circle.mode.ts:112]: marker on the kilometers

**Review by QuCMGisaia — APPROVED** [review]

File: projects/arlas-components/src/lib/components/mapgl/draw/modes/circles/circle.mode.ts (hunk 2 above)
Comment (mbarbet) [reply projects/arlas-components/src/lib/components/mapgl/draw/modes/circles/circle.mode.ts:36]: No, it's a parameter for the circle lib

**Review by MohamedHamouGisaia — APPROVED** [review]

**Pull request merged**